doc/developer-notes.md

```diff
@@ -857,12 +857,12 @@ Strings and formatting
     buffer overflows, and surprises with `\0` characters. Also, some C string manipulations
     tend to act differently depending on platform, or even the user locale.
 
-- Use `ParseInt32`, `ParseInt64`, `ParseUInt32`, `ParseUInt64`, `ParseDouble` from `utilstrencodings.h` for number parsing.
+- Use `ToIntegral` or `ParseInt32`, `ParseInt64`, `ParseUInt32`, `ParseUInt64`, `ParseDouble` from [`strencodings.h`](/src/util/strencodings.h) for number parsing. Or use `LocaleIndependentAtoi` which replicates `std::atoi` for backwards compatibility reasons.
```

Comment:
You could put this on a new line, rather than making this one ridiculously long.

Reply:
updated it.